src/_listByline.scss

```diff
@@ -37,7 +37,7 @@
       border-radius: $fd-list-byline-thumbnail-border-radius;
       margin-right: $fd-list-byline-thumbnail-spacing;
       font-size: 2.5rem;
-      align-self: flex-start;
+      align-self: center;
```

Comment:
I don't see this rule centering the icon horizontally. You still have the same:
<img width="312" alt="Screen Shot 2021-05-20 at 1 25 43 PM" src="https://user-images.githubusercontent.com/39598672/119022953-306eac80-b96f-11eb-9cd6-e177e3972f57.png">

You can use `text-align: center`

Reply:
you can observe in this example. (styles docs)
<img width="1434" alt="Screenshot 2021-05-20 at 11 04 14 PM" src="https://user-images.githubusercontent.com/57661487/119023693-e19d3300-b9bf-11eb-8e2d-5d01e517f52d.png">

Reply:
here Avatar is not centre aligned vertically.